R/wflow_use_github.R

```diff
@@ -410,14 +411,30 @@ create_gh_repo <- function(account, repository, account_is_organization = FALSE)
   # Create the repository
   message(glue::glue("Creating repository {repository}"))
   if (account_is_organization) {
-      req_create <- httr::POST(
-        glue::glue("https://api.github.com/orgs/{account}/repos"), token, ua,
-        body = list(name = repository), encode = "json"
+      req_create <- httr::RETRY(
+        verb = "POST"
+        , url = glue::glue("https://api.github.com/orgs/{account}/repos")
+        , config = token
+        , ua
+        , body = list(name = repository)
+        , encode = "json"
+        , terminate_on = c(
+          403, 404
+        )
```

Comment:
This is great! But could you please follow the style of the function call as it was?

Reply:
oh sure, sorry! fixed in https://github.com/jdblischak/workflowr/pull/199/commits/99036e145858ea2b9fb66749c571a270a5179e7f